windows-installer/fluent-template.conf

```diff
@@ -37,8 +37,17 @@
   channel application,system,security
   # The pos file is a place holder that should be updated
   # during the agent install process.
-  pos_file POS_FILE_PLACE_HOLDER
+  pos_file WIN_EVT_POS_FILE_PLACE_HOLDER
   # Read every 2 seconds.
   read_interval 2
   tag winevt.raw
 </source>
+
+<source>
+  type tail
+  format none
+  path CUSTOM_LOGS_DIR_PLACE_HOLDER
```

Comment:
Hmm, this directive will simply collect logs under the CustomLogs path. We want to give customers a way to augment their logging config instead to add such directives pointing to whatever path they choose.

Let's instead add the following at the top of the config:
```
@include config.d/*.conf
```
That way, users can create files under `$INSTDIR\config.d` with fluentd config directives.